arrocke/gloss-translation
# feat: format prisma pre-commit

## What has changed
- `nx run db:prisma format` is run before each commit. The formatting changes applied by this command are then added to the commit. 

## Connected Issues

closes #338

## QA Steps

1. Switch your local branch to this branch (`tyce-338-format-prisma`).
2. Locally, add spacing (tabs, spaces, etc.) to `schema.prisma` so that the file is not properly formatted.
   - If you have a VSCode extension which autoformats prisma files on save, you may need to disable that extension or temporarily change the "Language Mode" of `schema.prisma` from "Prisma" to "Plain Text" to test this PR.
3. Commit your changes.
4. Observe the diff of the commit. Confirm that there are no changes observed (since the formatting has been corrected).

## Post-Deployment
